CoffeeMachine_Task1/src/bg/coffee_machine/PremiumContainer.java

```diff
@@ -0,0 +1,17 @@
+package bg.coffee_machine;
+
+public class PremiumContainer extends Container {
+
+    public PremiumContainer() {
+        super(1000, 1000, 1000, 300);
+    }
+    public boolean enoughCappuccino(){
```

Comment:
This method does not seem to belong here. The same applies for `enoughMochaccino`. Consider moving them somewhere else. Container should not know what it costs to brew such a product or even that Container's supplies are used for coffee. They might be used for creating a building out of coffee beans and milk. If you remove them, then you will be able to use PremiumContainer for something completely different then CoffeeMachine.